src/main/java/WayofTime/alchemicalWizardry/AlchemicalWizardry.java

```diff
@@ -889,11 +898,17 @@ public void load(FMLInitializationEvent event)
 
         //Ore Dictionary Registration
         OreDictionary.registerOre("oreCoal", Blocks.coal_ore);
-        MeteorRegistry.registerMeteorParadigm(diamondStack, diamondMeteorArray, diamondMeteorRadius);
-        MeteorRegistry.registerMeteorParadigm(stoneStack, stoneMeteorArray, stoneMeteorRadius);
-        MeteorRegistry.registerMeteorParadigm(ironBlockStack, ironBlockMeteorArray, ironBlockMeteorRadius);
-        MeteorRegistry.registerMeteorParadigm(new ItemStack(Items.nether_star), netherStarMeteorArray, netherStarMeteorRadius);
-
+        MeteorRegistry.registerMeteorParadigm(diamondStack, Meteor00Array, Meteor00Radius);
+        MeteorRegistry.registerMeteorParadigm(stoneStack, Meteor01Array, Meteor01Radius);
+        MeteorRegistry.registerMeteorParadigm(ironBlockStack, Meteor02Array, Meteor02Radius);
+        MeteorRegistry.registerMeteorParadigm(new ItemStack(Items.nether_star), Meteor03Array, Meteor03Radius);
+        //add new meteor
+        //ItemStack BPBlock = GameRegistry.findItemStack("dreamcraft", "tile.BlackPlutonium", 1); 	//Use Black Plutonium block as catalyst to summon
+        //MeteorRegistry.registerMeteorParadigm(BPBlock, BPMeteorArray, BPMeteorRadius);			//continued
+        
+        ItemStack BPBlock = GT_ModHandler.getModItem("gregtech", "gt.blockores", 1, 6); 			//Use Lithium Ore as catalyst to summon
```

Comment:
I also had to use the GT_ModHandler here, since I couldn't figure out how to get GameRegistry.findItemStack to work with metadata, which is probably bad.

Reply:
I think you may just copy that function from GT, to avoid unnecessary dependency

Reply:
It isn't possible to check metadata with normal forge stuff? Not that I just couldn't figure it out?

Reply:
All getModItem() does is call GameRegistry.findItemStack, clones the result and applies Items.feather.setDamage.

Reply:
I added BMModHandler.java to do this, though I don't know if I need all those functions. And I don't know what a replacement item is either.